tests/workflow.go

```diff
@@ -1061,6 +1060,7 @@ func (s *FunctionalSuite) TestExecuteMultiOperation() {
 				s.NotZero(startRes.RunId)
 
 				updateRes := resp.Responses[1].Response.(*workflowservice.ExecuteMultiOperationResponse_Response_UpdateWorkflow).UpdateWorkflow
+				s.NotNil(updateRes.Outcome)
```

Comment:
I noticed during testing that the assertion below isn't sufficient (in the Update timeout case)